src/ember-app.js

```diff
@@ -492,20 +492,30 @@ class EmberApp {
       }
     }
 
-    debug('reading array of app file paths from manifest');
-    let appFiles = manifest.appFiles.map(function(appFile) {
-      return path.join(distPath, appFile);
-    });
+    let appFiles, vendorFiles, htmlFile, html;
 
-    debug('reading array of vendor file paths from manifest');
-    let vendorFiles = manifest.vendorFiles.map(function(vendorFile) {
-      return path.join(distPath, vendorFile);
-    });
+    if (manifest.htmlEntrypoint) {
```

Comment:
Is `ember-cli-fastboot` writing this into the manifest? or is it embroider?

Asking because we have a concept of `FastBootSchemaVersions` to ensure there is compatibility between the addon and this JS library.

Reply:
It's written by embroider.